test/tests/attachmentsTest.js

```diff
@@ -279,6 +279,29 @@ describe("Zotero.Attachments", function() {
 		});
 	});
 	
+	
+	describe("#importFromURL()", function () {
+		it("should download a PDF from a JS redirect page", async function () {
+			this.timeout(65e3);
+			
+			let stub = sinon.stub(Zotero.Attachments, 'downloadFile');
+			stub.callsFake(Zotero.Attachments.downloadPDFViaBrowser.bind(Zotero.Attachments));
```

Comment:
We should just hard-code our test URL above so we don't have to fake this (which probably caused this to miss the missing `await` in `downloadFile()`).